license-compliance/gradle-license-report/license-normalizer-bundle.json

```diff
@@ -159,6 +159,16 @@
       "bundleName": "unicode-and-icu",
       "licenseName": "Unicode/ICU",
       "licenseUrl": "https://github.com/unicode-org/icu/blob/release-72-1/icu4c/LICENSE"
+    },
+    {
+      "bundleName": "gpl2-with-foss1",
+      "licenseName": "GPL-2.0-with-Universal-FOSS-Exception-1.0",
+      "licenseUrl": "https://github.com/mysql/mysql-connector-j/tree/8.3.0?tab=License-1-ov-file"
```

Comment:
issue: It is incorrect to use the URL from GitHub for all `GPL-2.0-with-Universal-FOSS-Exception-1.0` licenses. This URL applies only to MySQL

Reply:
I found a separate license for GPL-2.0 and Universal-FOSS-Exception-1.0 but didn't find both. 
License report plugin finds `http://dev.mysql.com/doc/connector-j/en/` but it is incorrect too
I will try to find the whole license again

Reply:
Origin name of license found by License Report is `The GNU General Public License, v2 with Universal FOSS Exception, v1.0`
Maybe write two links into single licenseUrl, because preview of mysql license talk about general provisions and refer to the two licence.

Reply:
The reason it found this name is because this name is used in [build.xml](https://github.com/mysql/mysql-connector-j/blob/8.3.0/build.xml#L577) in [MySQL repo](https://github.com/mysql/mysql-connector-j/tree/8.3.0).
But the license for MySQL is located in its repo: https://github.com/mysql/mysql-connector-j/blob/8.3.0/LICENSE.
So, it is incorrect to use this link for general `The GNU General Public License, v2 with Universal FOSS Exception, v1.0`.

I think, as you suggested, we can add two URLs to the `licenseUrl` property: one for GPLv2 and one for Universal FOSS Exception. And remove the URL that point to the MySQL specific license